src/components/Elements/CardWithActions/CardItemWithActions.tsx

```diff
@@ -21,7 +21,7 @@ export const CardItemWithActions = ({
   sourceType = 'supported',
 }: CardItemWithActionsProps) => {
   const [showModal, setShowModal] = useState(false)
-  const [link, setLink] = useState({})
+  const [data, setData] = useState({})
```

Comment:
`data` is too generic, please use a name like `socialShareData` or `socialSharePayload`